### DISCUSSION TITLE
Donate link for craigbarratt@?

### DISCUSSION THREAD

**Discussion by j-steve** [custom-components/pyscript#427] [General]
I'm really loving Pyscript.  Is there a good way to donate to @craigbarratt for his work on building and maintaining the framework?

**Comment by ALERTua** [comment]
Pyscript changed my whole usage of Home Assistant. I don't even use the built-in automations: everything is performed by Pyscript. Without it, I would dump 90% of my automation ideas due to the clunky automation interface and YAML clunkiness.
I would love to show my appreciation for @craigbarratt's work.

**Comment by craigbarratt** [comment]
Thanks for the feedback!  I don't have a donate link, but your messages are much appreciated.
